pom.xml

```diff
@@ -103,10 +104,15 @@
                 <scope>import</scope>
             </dependency>
 
-            <!-- runtime dependencies -->
+			<!-- runtime dependencies -->
+			<dependency>
+				<groupId>org.ow2.asm</groupId>
+				<artifactId>asm</artifactId>
+				<version>${version.asm}</version>
+			</dependency>
```

Comment:
Minor details - formatting :) Can you adjust it to how it's done in other poms? I think we use 4 spaces instead of a tab.

Makes me wonder if we should add `.editorconfig`, but also look at how to automate it as part of the build (like in golang)

Reply:
Sorry, missed that. Fixed it.